### ISSUE TITLE
cloudresourcemanager.v3.getProjectIamPolicy returning 404 response

### ISSUE THREAD

**Issue by ColeSiegelTR** [pulumi/pulumi-google-native#172]
Version: 0.6.0

I am trying to update a project's IAM policy but when I try to retrieve the policy using `getProjectIamPolicy` I get 404 errors.

**Error Log**
```
Diagnostics:
  pulumi:pulumi:Stack (a206447_data-lake-common-infrastructure-int):
    Promise { <pending> }
 
    error: Error: invocation of google-native:cloudresourcemanager/v3:getProjectIamPolicy returned an error: error sending request: googleapi: got HTTP response code 404 with body: <!DOCTYPE html>
    <html lang=en>
      <meta charset=utf-8>
      <meta name=viewport content="initial-scale=1, minimum-scale=1, width=device-width">
      <title>Error 404 (Not Found)!!1</title>
      <style>
        *{margin:0;padding:0}html,code{font:15px/22px arial,sans-serif}html{background:#fff;color:#222;padding:15px}body{margin:7% auto 0;max-width:390px;min-height:180px;padding:30px 0 15px}* > body{background:url(//www.google.com/images/errors/robot.png) 100% 5px no-repeat;padding-right:205px}p{margin:11px 0 22px;overflow:hidden}ins{color:#777;text-decoration:none}a img{border:0}@media screen and (max-width:772px){body{background:none;margin-top:0;max-width:none;padding-right:0}}#logo{background:url(//www.google.com/images/branding/googlelogo/1x/googlelogo_color_150x54dp.png) no-repeat;margin-left:-5px}@media only screen and (min-resolution:192dpi){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat 0% 0%/100% 100%;-moz-border-image:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) 0}}@media only screen and (-webkit-min-device-pixel-ratio:2){#logo{background:url(//www.google.com/images/branding/googlelogo/2x/googlelogo_color_150x54dp.png) no-repeat;-webkit-background-size:100% 100%}}#logo{display:inline-block;height:54px;width:150px}
      </style>
      <a href=//www.google.com/><span id=logo aria-label=Google></span></a>
      <p><b>404.</b> <ins>That’s an error.</ins>
      <p>The requested URL <code>/v3/projects/a465bd909a63:getIamPolicy?alt=json</code> was not found on this server.  <ins>That’s all we know.</ins>
    
        at Object.callback (/Users/colesiegel/Documents/Workspaces/a206447_data-lake-common-infrastructure/node_modules/@pulumi/pulumi/runtime/invoke.js:140:33)
        at Object.onReceiveStatus (/Users/colesiegel/Documents/Workspaces/a206447_data-lake-common-infrastructure/node_modules/@grpc/grpc-js/src/client.ts:338:26)
        at Object.onReceiveStatus (/Users/colesiegel/Documents/Workspaces/a206447_data-lake-common-infrastructure/node_modules/@grpc/grpc-js/src/client-interceptors.ts:426:34)
        at Object.onReceiveStatus (/Users/colesiegel/Documents/Workspaces/a206447_data-lake-common-infrastructure/node_modules/@grpc/grpc-js/src/client-interceptors.ts:389:48)
        at /Users/colesiegel/Documents/Workspaces/a206447_data-lake-common-infrastructure/node_modules/@grpc/grpc-js/src/call-stream.ts:276:24
        at processTicksAndRejections (node:internal/process/task_queues:76:11)
```

I have verified the following:
- I am able to get the project policy using CLI `gcloud projects get-iam-policy a465bd909a63`
- I have the permission `resourcemanager.projects.getIamPolicy`
- I am able to use google-native `getProject` with the same parameters and I get a correct response

## Steps to reproduce

```
import * as gcp from "@pulumi/google-native";
const policy = gcp.cloudresourcemanager.v3.getProjectIamPolicy({
    project: `a465bd909a63`
});
console.log(policy);
```

Expected: Get the policy
Actual: 404

## Question

Maybe I am just passing the parameters incorrectly, but it appears project-id should work, please advise on the usage or if this is a bug. Seems related to https://github.com/pulumi/pulumi-google-native/pull/115. I also tried the earlier versions (v1) without any luck.

**Comment by viveklak** [comment]
This is because the HTTP verb is being assumed to be a GET but in fact the corresponding endpoint requires a POST. Fix in #175

**Comment by ColeSiegelTR** [comment]
Thanks for the quick fix @viveklak ! Is there a way to tell which version will have the fix? Or will it just be whenever the next release happens to fall?

**Comment by viveklak** [comment]
This should be available in 0.7.0. We will release that in the next day or two.

**Comment by ColeSiegelTR** [comment]
Hi @viveklak, I realized this is happening for some other methods too, such as 
`cloudkms.v1.getKeyRingIamPolicy`. Would #175 resolve that or should I raise a separate issue?

**Comment by viveklak** [comment]
> Hi @viveklak, I realized this is happening for some other methods too, such as
> `cloudkms.v1.getKeyRingIamPolicy`. Would #175 resolve that or should I raise a separate issue?

I didn't see any changes for `getKeyRingIamPolicy` based on the updated metadata generated from the fix in #175. Would you be able to open a separate issue for what you are seeing for `getKeyRingIamPolicy`?
